README.md

```diff
@@ -108,7 +94,7 @@ The end of the output will include important performance summary metrics such as
 
 #### 4. Use the Results
 
-The results from GuideLLM are used to optimize your LLM deployment for performance, resource efficiency, and cost. By analyzing the performance metrics, you can identify bottlenecks, determine the optimal request rate, and select the most cost-effective hardware configuration for your deployment.
+The results from GuideLLM are used to enhance your LLM deployment for performance, resource efficiency, and cost. By analyzing the performance metrics, you can identify bottlenecks, determine the optimal request rate, and select the most cost-effective hardware configuration for your deployment.
```

Comment:
optimize/enhance, perhaps `guide enhancement of your LLM deployment` ... ? I am not clear how guidellm directly enhances a deployment, unless we plan to add some type of automated management here.